### ISSUE TITLE
Can not open pkcs12 file generated by sample

### ISSUE THREAD

**Issue by vovkvlad** [PeculiarVentures/PKI.js#105]
Hi guys,
I am trying to open `.p12` file generated by your "PKCS12SimpleExample". During opening, it asks for a password, after entering password I used for generating this file, it says incorrect password.

When I try to open file generated using "With Certificate-Based Integrity Protection And No Privacy Protection" variant it still asks me for a password, and here I have no clue which password should I enter.
Please see attached files for more info.

If I am doing something wrong - please let me know,
Thanks in advance

![image](https://cloud.githubusercontent.com/assets/6311961/25668574/bb4b4f48-302f-11e7-946b-823c53f5af11.png)
![image](https://cloud.githubusercontent.com/assets/6311961/25668597/c81aaf2a-302f-11e7-93d8-4a9e6a76fb5c.png)

**Comment by YuryStrozhevsky** [comment]
Reason why you have the issue is that PKI.JS uses advanced algorithms but current modern OSes do not. Later Ryan could send you a link to article with more explanation. At the same time you could search for same closed issues in PKI.JS repo.

**Comment by rmhrisk** [comment]
@vovkvlad more likely than not OSX only supports weak ciphers for the P12, see http://unmitigatedrisk.com/?p=543 for some background.

If my guess is right to produce a file that it will open would require us to do these items: https://github.com/PeculiarVentures/PKI.js/issues/101

We have held off doing them because we have tried taking a position native and modern crypto is the right way to build things. We created #101 because we have come to the point we accept UAs may not be updating anytime soon which means it may be needed for some people.

We have no plan for when we will get this work done, coordinated PRs are welcome.
